tests/conftest.py

```diff
@@ -57,3 +58,18 @@ def fake_fs():
         patcher.fs.add_real_directory(test_data_path)
 
         yield patcher.fs # This will return a fake file system
+
+
+def set_home(monkeypatch, path):
+    if platform.system() == 'Windows':
+        monkeypatch.setenv('USERPROFILE', __tests__ + path)
+    else:
+        monkeypatch.setenv('HOME', __tests__ + path)
+
+
+def set_az_path(monkeypatch):
+    if platform.system() == 'Windows':
+        monkeypatch.setenv('Path', __tests__ + "\\testdata\\windows\\")
+        monkeypatch.setenv('COMSPEC', 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe')
```

Comment:
I'm not familiar with this variable, what is it used for? Do we only need to set this environment variable when configuring Path?

Reply:
This basically points to the command line interpreter by default windows program run on cmd. But we are running a powershell script to mock the `az` so we need to run it on powershell.

Reply:
Ah understood, and this is only needed for tests that run our mock `az` command.